test_settings:
  seed: 40
  base_file_name: "td7_agent"
  save_date: "20240811"
  max_episode_steps: 500
  use_checkpoints: True
  num_of_test_episodes: 4 # Should match with the start-goal pairs below

start_goal_pairs:
- start: {x: -8.0, y: -8.0, theta: 0}
  goal: {x: 8.0, y: 8.0}
- start: {x: -8.0, y: 8.0, theta: 0}
  goal: {x: 8.0, y: -8.0}
- start: {x: -8.0, y: 3.0, theta: 0}
  goal: {x: 8.0, y: -3.0}
- start: {x: -3.0, y: -8.0, theta: 0}
  goal: {x: 3.0, y: 8.0}
